File: testdata/bench/perl/BenchmarkTest00004.pl
use Mojolicious::Lite;
get '/search' => sub {
    my $c = shift;
    my $query = $c->param('q');
    my $dbh = DBI->connect("dbi:Pg:dbname=app", "", "");
    my $sth = $dbh->prepare("SELECT * FROM items WHERE title LIKE ?");
    $sth->execute("%$query%");
};
app->start;

File: testdata/bench/lua/BenchmarkTest00004.lua
local pgmoon = require "pgmoon"
local args = ngx.req.get_uri_args()
local id = args.id
local pg = pgmoon.new({host="127.0.0.1", database="app"})
pg:connect()
local res = pg:query("SELECT * FROM orders WHERE id = $1", id)
ngx.say(res)
